mavros/src/plugins/global_position.cpp

```diff
@@ -450,6 +453,16 @@ class GlobalPositionPlugin : public plugin::PluginBase {
 		map_origin.y() = req->geo.longitude;
 		map_origin.z() = req->geo.altitude;
 
+		/**
+		 * @brief Conversion from geodetic coordinates (LLA) to ECEF (Earth-Centered, Earth-Fixed)
+		 */
+		GeographicLib::Geocentric map(GeographicLib::Constants::WGS84_a(),
```

Comment:
Please do it the same way it is done in `fake_gps`: https://github.com/mavlink/mavros/blob/master/mavros_extras/src/plugins/fake_gps.cpp#L85-L98